waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -97,6 +98,17 @@ alexGetByte (_, [], (currChar:remainingSource)) = case encodeChar currChar of
 alexInputPrevChar :: AlexInput -> Char
 alexInputPrevChar (prevChar, _, _) = prevChar
 
+-- | Tokenizes the given string. 
+alexScanTokens :: String -> Maybe [Parser Token]
+alexScanTokens str = go ('\n', [], str)
+  where
+    go input@(_, _, lexeme) = case alexScan input 0 of
+      AlexEOF -> Just []
+      AlexError _ -> Nothing
+      AlexSkip rest _ -> go rest
```

Comment:
What is rest here? Rest of input? Let's name it `inputRest` then, or `restOfInput`. Same thing below.

Reply:
I've been using Haskell for 3 weeks and have already become scared of normal variable names :upside_down_face: 

`restOfInput` it is.